query/src/main/java/io/keen/client/java/Query.java

```diff
@@ -36,8 +42,8 @@
      *
      * @return The JSON object map.
      */
-    public Map<String, Object> constructQueryArgs() {
-
+    @Override
+    Map<String, Object> constructRequestArgs() {
```

Comment:
Comment: Again, technically changing a `public` method to be non-public is a breaking change. In this case it's pretty hard to imagine anybody calling this in client code, but you never know...